File: Walsh_A6_PHP/entity/Car.php
<?php
/**
 * Description of Car
 * Car is a PHP class that defines the key fields for storing and creating a Car Obj
 * @author Brady Walsh
 * @param $ID - The ID number for the car, also the PK for the Database     DataType: String
 * @param $year - The Year the car was made                                 DataType: Integer <---- only one Integer
 * @param $make - The Manufacturer of the Car (ie. Ford, Tesla, etc)        DataType: String
 * @param $model - The Model, or "Type" of car.                             DataType: String
 * @param $colour - The Colour of the car.                                  DataType: String 
 */
class Car implements JsonSerializable{
    private $year;
    private $make;
    private $model;
    private $colour;
    
    //Public Constructor for Car Objects
    public function __construct( $inYear, $inMake, $inModel, $inColour) {
        $this->year = $inYear;
        $this->make = $inMake;
        $this->model = $inModel;
        $this->colour = $inColour;
    }
    
    //Getters
    
    
    public function getCarYear(){
        return $this->year;
    }
    public function getCarMake(){
        return $this->make;
    }
    public function getCarModel(){
        return $this->model;
    }
    public function getCarColour() {
        return $this->colour;
    }
    
    //Setters
    
    public function setCarID($newID){
        $this->ID = $newID;
    }
    public function setCarYear($newYear){
        $this->year = $newYear;
    }
    public function setCarMake($newMake) {
        $this->make = $newMake;
    }
    public function setCarModel($newModel){
        $this->model = $newModel;
    }
    public function setCarColour($newColour){
        $this->colour = $newColour;
    }
    
    //Methods
    
    public function isItAClassic(){
        $check = false;
        if($this->year < 1992)
        {$check = true;}
        return $check;
    }
    
    public function isItAnAntique(){
        $check = false;
        if($this->year < 1972)
        {$check = true;}
        return $check;
    }
    
    public function jsonSerialize() {
        return get_object_vars($this);
    }

}

File: Walsh_A3_AJD/src/java/entity/Car.java
package entity;

/**
 *
 * @author brady
 */
public class Car {
    private int Year;
    private String Make;
    private String Model;
    private String Colour;
    
    public Car(int inYear, String inMake, String inModel, String inColour){
        this.Year = inYear;
        this.Make = inMake;
        this.Model = inModel;
        this.Colour = inColour;
        
    }
        //Getters
    
    
    public int getCarYear(){
        return Year;
    }
    public String getCarMake(){
        return Make;
    }
    public String getCarModel(){
        return Model;
    }
    public String getCarColour() {
        return Colour;
    }
    
    //Setters
    
    public void setCarYear(int newYear){
        this.Year = newYear;
    }
    public void setCarMake(String newMake) {
        this.Make = newMake;
    }
    public void setCarModel(String newModel){
        this.Model = newModel;
    }
    public void setCarColour(String newColour){
        this.Colour = newColour;
    }
    
}
